sources/PTYTextView.m

```diff
@@ -6394,6 +6407,24 @@ - (void)setNeedsDisplayOnLine:(int)y inRange:(VT100GridRange)range {
     [self setNeedsDisplayInRect:dirtyRect];
 }
 
+- (void)setNeedsDisplayOnCol:(int)x inRange:(VT100GridRange)range {
+    NSRect dirtyRect;
```

Comment:
This can simply do `[self setNeedsDisplay:YES]` because it's unsafe to draw less than a whole row at a time.

Reply:
Done.